compiler/test-resources/repl/i15493

```diff
@@ -5,7 +5,7 @@ scala> NInt(23)
 val res0: NInt = NInt@17
 
 scala> res0.toString
-val res1: String = NInt@17
+val res1: String = "rs$line$1$NInt@17"
```

Comment:
Not sure what's happening here, but there must be some super sketchy stdout-regexing happening to mangle the `.toString` so it looks different when `return`ed or `println`-ed. 

The new behavior is probably better: we special case `return`ing because it uses `pprint`, `println` is just `println`, and if someone wants `pprint` themselves they can use `dotty.shaded.pprint.log`

```scala
scala> res0.toString
val res1: String = NInt@17
                                                                                                                          
scala> println(res0.toString)
rs$line$1$NInt@17
```